src/Test.UnitTests.Sarif/GitInformationTests.cs

```diff
@@ -1,42 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using FluentAssertions;
-using Moq;
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.Sarif
-{
-    public class GitInformationTests
-    {
-        [Fact]
-        public void GetRepositoryRoot_WhenDotGitIsAbsent_ReturnsNull()
```

Comment:
>public [](start = 8, length = 6)

Nothing was removed. The rename + the additional tests made Git decide that the old `GitInformationTest.cs` and the new `GitHelperTests.cs` were different files.